app/cito_engine/models.py

```diff
@@ -194,25 +194,30 @@ def update_counters(self, timer_value):
 
     @property
     def is_action_required(self):
+        # Got an incident outside the timer
+        if self.timer > self.event_action.threshold_timer:
+            # If threshold was one incident in Y secs, we have to return True
+            if self.event_action.threshold_count == 1:
+                self.is_triggered = True
+                self._reset_all()
+                return True
+
+            # else
+            self.is_triggered = False
+            self._reset_all()
+
```

Comment:
@evelynleems for sake of better code readability, please have L206-207 within an `else:` statement